libs/blocks/global-navigation/global-navigation.js

```diff
@@ -121,14 +130,33 @@ const setupKeyboardNav = async () => {
   });
 };
 
-// TODO - when clicking the navigation the dropdowns currently do not close.
-const closeOnClickOutside = (e) => {
-  if (
-    !e.target.closest(selectors.globalNav)
-    || e.target.closest(selectors.curtain)
-  ) {
-    closeAllDropdowns();
+const getBrandImage = (image) => {
+  // Return the default Adobe logo if an image is not available
+  if (!image) return CONFIG.icons.company;
+
+  try {
+    // Try to decorate image as SVG
+    const decoratedSvg = decorateSVG(image);
+    // 'decorateSVG' might return the original element if decoration fails
+    // or the picture wrapped in an anchor element in certain cases
+    const svg = decoratedSvg instanceof HTMLPictureElement
+      ? decoratedSvg : decoratedSvg.querySelector('picture');
+    if (svg) return svg;
+  } catch (e) {
```

Comment:
`decorateSVG` can return either the original anchor element or a decorated anchor element with a `picture` child element or a `picture` element directly, so this is the safest option as opposed to performing checks every step of the way.

Reply:
But `decorateSVG` would never throw an error, just return al element, so you'd never get in the catch block.
What am I missing here? 🤔

Reply:
It can return either an `a` tag or a `picture` one. `const svg = decoratedSvg instanceof HTMLPictureElement ? decoratedSvg : decoratedSvg.querySelector('picture');` would fail if we get back the original anchor. Instead of performing a series of checks, since we're using an external method which might change anyway, I preferred to go for the best case scenario and wrap it in a `try/catch`